openssl-dynamic/src/main/c/sslutils.c

```diff
@@ -555,8 +554,7 @@ int SSL_CTX_use_certificate_chain(SSL_CTX *ctx, const char *file,
     return n;
 }
 
-int SSL_CTX_use_certificate_chain_bio(SSL_CTX *ctx, BIO *bio,
-                                  int skipfirst)
+static int SSL_CTX_setup_certs(SSL_CTX *ctx, BIO *bio, int skipfirst, int ca)
```

Comment:
@Scottmitch I think I would like to keep it as it is and cleanup all the methods in a followup PR as this should be done for all the methods in all classes where we use an int as long. Ok ?